src/backend/opencl/cpu/cpu_sparse_blas.cpp

```diff
@@ -0,0 +1,530 @@
+/*******************************************************
+ * Copyright (c) 2014, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#if defined(WITH_OPENCL_LINEAR_ALGEBRA)
```

Comment:
How would you deal with it in the source code that calls functions from this file ?